src/components/LabelGroup/component.tsx

```diff
@@ -13,16 +14,30 @@ class LabelGroup extends BaseComponent <LabelGroupArgs, any> {
 
     attachElement = (nodeElement: HTMLElement, containerElement: any) => {
         if (!nodeElement) return;
+        const childrenErrorMessage = 'A LabelGroup must contain a single PCUI react component as a child';
+        // check that the LabelGroup has a single child
         if (Array.isArray(this.props.children) || !this.props.children) {
-            throw new Error('A LabelGroup must contain a single child react component');
+            throw new Error(childrenErrorMessage);
+        }
+        // casting child as a single ReactElement as we have confirmed it is above
+        const child = this.props.children as ReactElement;
```

Comment:
Why is the name of the property `children` if it's a single child?

Reply:
This children prop is provided by react and includes any child react elements a user has added to their LabelGroup. It's then been limited to a single react element above this line.